src/ChunkMap.cpp

```diff
@@ -2593,6 +2593,33 @@ bool cChunkMap::ForEachChunkInRect(int a_MinChunkX, int a_MaxChunkX, int a_MinCh
 
 
 
+void cChunkMap::ForEachLoadedChunk(std::function<bool(int, int)> a_Callback)
```

Comment:
All our `ForEach` functions return a bool, specifying if they enumerated everything, or the callback aborted. You should use it, too.